tests/snapshots/tests.contrib.django.test_django_snapshots.test_psycopg_query_default.snap

```diff
@@ -1,38 +1,20 @@
 [[{"name" "postgres.query"
    "service" "postgresdb"
-   "resource" "select 'one' as x"
+   "resource" "select 'one' as \"x\""
    "type" "sql"
    "error" 0
    "span_id" 0
    "trace_id" 0
    "parent_id" nil
-   "start" 1623177658188007300
-   "duration" 1828500
-   "meta" {"runtime-id" "98e7426d5d6d44d1a5d76d1322d69cac"
+   "start" 1624103334938617300
+   "duration" 1192800
+   "meta" {"runtime-id" "25fe3bf994c8480dbf2e18d4ea25956b"
            "django.db.vendor" "postgresql"
            "django.db.alias" "postgres"}
    "metrics" {"_dd.agent_psr" 1.0
               "_dd.measured" 1
               "_sampling_priority_v1" 1
-              "system.pid" 236
+              "system.pid" 2152
               "db.rowcount" 1
               "sql.rows" 1
-              "_dd.tracer_kr" 1.0}}
-      {"name" "postgres.query"
-       "service" "postgres"
-       "resource" "select 'one' as x"
-       "type" "sql"
-       "error" 0
-       "span_id" 1
-       "trace_id" 0
-       "parent_id" 0
-       "start" 1623177658188405100
-       "duration" 1265000
-       "meta" {"out.host" "127.0.0.1"
-               "db.name" "test_postgres"
-               "db.user" "postgres"
-               "db.application" "None"}
```

Comment:
with these changes we collapsed the two spans together... but we lost this metadata...

Reply:
maybe instead of `pin.onto(cursor.cursor)` we should try to merge the data from both pins?

```python
old_pin = Pin.get_from(cursor.cursor)
pin.tags.update(old_pin.tags)
pin.onto(cursor.cursor)
```


we might also need to do something like:

```python
cursor.cursor._self_datadog_name = "{}.query".format(pin.app)
```

It seems like this data is set on initialization of the traced cursor, so we might need to update it?